src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  --hover-active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.header__img_logo {
+  margin: 10px 0;
+  cursor: pointer;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  list-style: none;
+  display: flex;
+}
+
+.nav__item:nth-of-type(n + 2) {
```

Comment:
You could also do here `.nav__item:not(:first-of-type)` but your solution still works

Reply:
less math less problems 😂